### ISSUE TITLE
Add Softmax on top of a pre-trained model

### ISSUE THREAD

**Issue by mayuanyang** [Microsoft/CNTK#934]
Hi, 
I just download the pre-trained cifar10 model which hasn't got a Softmax classfier , can i add one on top of it without actually retrain the whole model?
Thanks

**Comment by avader906** [comment]
You can express the underlying model in BrainScript in its linear algebra form and then add on a layer

https://github.com/Microsoft/CNTK/wiki/BS-Expressions#expression-types

**Comment by wolfma61** [comment]
Using CNTK from Brainscript has been moved here: https://docs.microsoft.com/en-us/cognitive-toolkit/Using-CNTK-with-BrainScript
